lib/yotpo/core/response_parser.rb

```diff
@@ -19,7 +19,7 @@ class ParseOj < FaradayMiddleware::ResponseMiddleware
     end
 
     define_parser do |body|
-      ::Oj.load(body, mode: :compat) unless body.strip.empty?
+      ::Oj.load(body, mode: :compat) unless body.to_s.strip.empty?
```

Comment:
@threadhead  i would use "if body.present? && body.strip.present?" instead of going through "to_s" and this "unless"